apps/condo/next-env.d.ts

```diff
@@ -1,2 +1,3 @@
 /// <reference types="next" />
 /// <reference types="next/types/global" />
+declare module '*.md'
```

Comment:
Why do you need this declaration? It's a bit hacky...

Why not using approach which used before when dealing with md/mdx files?

1. Extract file content in `getStaticProps/getServerSideProps`
2. Pass it as input to `@open-condo/ui` Markdown component.

Reply:
> 1. Extract file content in getStaticProps/getServerSideProps
Done